cli/client.go

```diff
@@ -380,6 +387,83 @@ func RobotsPartShellAction(c *cli.Context) error {
 	)
 }
 
+// CheckUpdateResponse holds the values used to hold release information.
+type CheckUpdateResponse struct {
+	Name       string `json:"name"`
+	TagName    string `json:"tag_name"`
+	TarballURL string `json:"tarball_url"`
+}
+
+// CheckUpdate is the corresponding Action for 'check-update'.
+func CheckUpdate(c *cli.Context) error {
+	if c.Bool(quietFlag) {
+		return nil
+	}
+
+	infof(c.App.ErrWriter, "checking if update available")
```

Comment:
Necessary? I had imagined there would be no output if an update was not "available".

Reply:
the extra infos have been removed now, only printing the warning when out of date